engine/gamesys/src/gamesys/components/comp_label.h

```diff
@@ -53,7 +53,12 @@ namespace dmGameSystem
 
     const char* CompLabelGetText(const LabelComponent* component);
 
-
+    Vectormath::Aos::Matrix4 CompLabelLocalTransform(
+        const Vectormath::Aos::Point3& position,
+        const Vectormath::Aos::Quat& rotation,
+        const Vectormath::Aos::Vector3& scale,
+        const Vectormath::Aos::Vector3& size,
+        uint32_t pivot);
```

Comment:
Same as above, arguments on same row.

Reply:
Put everything on the same line. Thanks! :)